terraform/templates/ec2_user_data.sh

```diff
@@ -34,18 +34,17 @@ cat > /etc/profile.d/libra_prompt.sh <<EOF
 export PS1="[\u:validator@\h \w]$ "
 EOF
 
-if ${enable_logrotate}; then
-	cat >> /etc/logrotate.conf <<"EOF"
-	${log_path} {
-		daily
-		rotate 100
-		compress
-		delaycompress
-	}
-	EOF
-
-	sudo logrotate /etc/logrotate.conf
-fi
+{% if enable_logrotate %}
+cat > /etc/logrotate.d/libra <<EOF
+${log_path} {
+	daily
+	size 100M
```

Comment:
Can we remove size? Or set it to something bigger like 10G?
We produce a lot of logs in cluster test, log rotating it frequently will make debugging hard

Reply:
Dev filled the entire disk in a day, this needs a limit.

Reply:
The rotated files should have timestamps, it should be easy enough to collect the log files for a certain day.

Reply:
Can we make it bigger then? What is size of dev disk? Surely its not 100M

Reply:
Alternatively, can we just make it configurable?

Reply:
You're welcome to make a diff to make this configurable, right now I just want something sane so dev doesn't go down everyday.